source/ui/BioXAS/BioXASPersistentView.cpp

```diff
@@ -38,6 +38,13 @@ BioXASPersistentView::BioXASPersistentView(QWidget *parent) :
 
 	setLayout(layout);
 
+	// Create SR1 current view.
+
+	BioXASControlEditor *sr1CurrentEditor = new BioXASControlEditor(CLSStorageRing::storageRing()->ringCurrentControl());
+	sr1CurrentEditor->setTitle("SR1 Current");
```

Comment:
I checked, because I know we've had issues with this in the past. This control editor is read-only. :)